aspnetcore/razor-components/components.md

```diff
@@ -54,21 +54,21 @@ In the following example, the `ParentComponent` sets the value of the `Title` pr
 
 *ParentComponent.cshtml*:
 
-[!code-cshtml[](common/samples/3.x/BlazorSample/Pages/ParentComponent.cshtml?start=1&end=7&highlight=5)]
+[!code-cshtml[](common/samples/3.x/BlazorSample/Pages/ParentComponent.cshtml?start=1&end=8&highlight=5)]
```

Comment:
Consider using a snippet to replace start and end line numbers.

Reply:
I did. :smile: ... lazy bones Rex thought he would do it on Preview 3. No, no ... I do want to do this now. I'll get it on the next commit.

Reply:
OH! ... now I know why ... no, I wasn't being lazy. That would require the snippet to start on Line 1 instead of the directive, and IIRC the app chokes on it (or did choke on me once) when I tried it.

Do you know if it will work? ... or should I try again and see what happens.

Reply:
Hold on ... I'll check. If it breaks, I'll move this bit to a snapshot file for the topic.

Reply:
:+1: Looks like it was a point-in-time thing.